snippets/card-product.liquid

```diff
@@ -498,8 +498,16 @@
                     </div>
                     <div>
                       <div class="quick-add-modal__content-info--bulk-details small-hide medium-hide">
-                        <h3 class="h2">{{ card_product.title | escape }}</h3>
-                        {% render 'price', product: card_product, price_class: '', show_compare_at_price: true %}
+                        <a
+                          href="{{ card_product.url }}"
+                          id="ProductTitleLink-{{ section_id }}-{{ card_product.id }}"
```

Comment:
I wonder if this ID is needed for anything 🤔 I don't think so.

Reply:
That's a good point, if we're not using it I might as well not include it. I'll remove.